saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/nlp/EntityRelation/EntityRelationClassifiers.scala

```diff
@@ -17,22 +17,22 @@ object EntityRelationClassifiers {
   object OrganizationClassifier extends Learnable(tokens) {
     def label: Property[ConllRawToken] = entityType is "Org"
     override lazy val classifier = new SparseNetworkLearner()
-    override def feature = using(word, windowWithin[ConllRawSentence](EntityRelationDataModel, -2, 2, List(pos)), phrase,
+    override def feature = using(word, tokensWithinWindowPos, phrase,
```

Comment:
I was getting an error when using the original `propertyWithin` features. So I tried out this changed from the other PR and the property worked fine.